wire-ios-utilities/Source/Public/DispatchGroupQueue.swift

```diff
@@ -26,15 +26,15 @@ public final class DispatchGroupQueue: NSObject, ZMSGroupQueue {
 
     public init(queue: DispatchQueue) {
         self.queue = queue
-        self.dispatchGroupContext = DispatchGroupContext(groups: [])
+        dispatchGroupContext = DispatchGroupContext(groups: [])
     }
 
-    public  var dispatchGroup: ZMSDispatchGroup! {
-        return self.dispatchGroupContext.groups.first
+    public  var dispatchGroup: ZMSDispatchGroup {
+        dispatchGroupContext.groups[0]
```

Comment:
why the change ? shouldn't we keep first and have it optional?

Reply:
The implicit force-unwrap results from an unspecified nullability in Objective C code. Since we try not to use force-unwraps when possible we should either declare `dispatchGroup` as optional or not. 
I will have another look.

Reply:
Fixed optionality in the protocol and where needed: 068fb1c9f59272bde53e8a0f84cc86506c6d1752